testing/testcafe/tests/scheduler/layout/appointments/dataSource.ts

```diff
@@ -20,4 +20,32 @@ test.skip('Appointment key should be deleted when removing an appointment from s
 
     .expect(compareResults.isValid())
     .ok(compareResults.errorMessages());
+}).before(async () => {
+  const initScheduler = ClientFunction(() => {
+    const devExpress = (window as any).DevExpress;
+
+    ($('#container') as any).dxScheduler({
+      dataSource: new devExpress.data.DataSource({
```

Comment:
don't we have another way to get access to `DataSource`?

Reply:
No, I don't know another way. This approach uses wide in our tests.